packages/techdocs-cli/src/commands/migrate/migrate.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { SingleHostDiscovery } from "@backstage/backend-common";
+import { Publisher } from "@backstage/techdocs-common";
+import { Command } from "commander";
+import { createLogger } from "../../lib/utility";
+import { getValidPublisherConfig } from "../helpers"
+
+export default async function migrate(cmd: Command) {
```

Comment:
Are we planning to write a test for this new function?

Reply:
We look at testing this and kind of decided against it, since the test would be so heavily tied to the implementation (lots of mocking/stubbing).